sdk/pinocchio/src/account_info.rs

```diff
@@ -430,17 +436,45 @@ impl AccountInfo {
         // - 8 bytes for the data_len
         //
         // So we can zero out them directly.
+        #[cfg(target_os = "solana")]
+        sol_memset_(self.data_ptr().sub(48), 0, 48);
```

Comment:
you can use slice::fill and can get rid of the cfg attr

Reply:
Using `slice::fill` takes `58` CUs – same as the current code. The `sol_memset_` is more efficient in this case.

Reply:
yes, but why?

If the compiler is desugaring to stores, CUs should be lower than the syscall - it's the whole point of applying the optimization. It might mean that the optimization is wrong, or the syscall is mispriced.

Reply:
I had the same question. 😊 @LucasSte looked into it and already has a plan to improve it.

Reply:
Four stores is the threshold to call `memset` from compiler-builtins. This is what causing the CU bump. Both the threshold to call the library in LLVM and the threshold to call the syscall in compiler-builints needs to be revised.